_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -9,15 +9,14 @@
 
 # Sometimes the script needs help to establish where it is in the file system
 SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-
 cd "$SCRIPT_DIR"
 
 ####################################
 # INITIALISE VARIABLES
 ####################################
 
 # Set % threshold for translated .md files to be created
-THRESHOLD="80"
+THRESHOLD="0"
```

Comment:
This has been set to 0 because if higher, some untranslated documents (from newer EN .md files) prevent the site from showing in that language. It's because of some dependency between .md files.

Reply:
Changed it back to 80 since 'release' is not affected by the .md file dependencies yet.